keyboards/yandrstudio/zhou65/readme.md

```diff
@@ -0,0 +1,16 @@
+zhou65 version 1.0
+===
+
+A 65% keys keyboard with rgb (keys).
+This keyboard use 8mhz HSE and STM32F401 as MCU.
+
+Keyboard Maintainer: https://github.com/jiaxin96
+Hardware Supported: zhou65
+Hardware Availability: https://github.com/Oh-My-Mechanical-Keyboard 
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```